### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by kalavhan**
First pull request for the review of the project "Forms" of the Ruby on rails course

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <%- text_field_tag(:username) -%>
+
+  <%- label_tag(:email, 'Email') -%>
+  <%- email_field_tag(:email) -%>
+
+  <%- label_tag(:password, 'Password') -%>
+  <%- password_field_tag(:password) -%>
+
+  <%- submit_tag("Search") -%>
+<%- end -%>
```
Comment (samgaco) [inline app/views/users/edit.html.erb:31]: - [ ] form_tag should be commented out too, we only keep the form_for

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  <%- text_field_tag(:username) -%>
+
+  <%- label_tag(:email, 'Email') -%>
+  <%- email_field_tag(:email) -%>
+
+  <%- label_tag(:password, 'Password') -%>
+  <%- password_field_tag(:password) -%>
+
+  <%- submit_tag("Search") -%>
+<%- end -%>
```
Comment (samgaco) [inline app/views/users/new.html.erb:31]: - [ ] form_tag should be commented out too, we only keep the form_for :+1:

**Review by samgaco — APPROVED** [review]
Hi @kalavhan and @fegzycole :high_brightness: 

Everything is perfect, the project is approved :white_check_mark: The only thing left is to comment out the form_tags, since there is no necessity to have more than one form field, the form_for should be the only one being used and the rest commented. Since it is a quick fix, please make that commit and you can move on :+1: 

------------------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

File: app/views/users/new.html.erb (hunk 2 above)
Comment (fegzycole) [reply app/views/users/new.html.erb:31]: Hello @samgaco, thank you for your feedback. If you open the edit.html.erb/new.html.erb files in your browser, you will see that they have actually been commented out.

Best Regards

**Pull request merged**